src/components/Table/TableBody/TableBodyRow/TableBodyRow.jsx

```diff
@@ -419,6 +419,8 @@ const TableBodyRow = ({
             data-offset={offset}
             offset={offset}
             width={matchingColumnMeta && matchingColumnMeta.width}
+            align={matchingColumnMeta && matchingColumnMeta.align}
+            className={col.isSortable === true ? `data-table-${matchingColumnMeta.align}` : ``}
```

Comment:
is there a reason we only do this if the row is sortable?

Reply:
Alignment class added in both case 1. With sorting icon 2. Without sorting icon

Reply:
Code is updated based on your review comments, Please review.